tests/Functional/Auth/SynapseUserReflectionTest.php

```diff
@@ -10,8 +10,8 @@
 
 class SynapseUserReflectionTest extends SynapseBaseCase
 {
-    private const LOGIN_PREFIX = 'UTILS_TEST_AUTH_LOGIN_';
-    private const LOGIN_PASSWORD = 'Str0ngPassword!';
+    protected const LOGIN_PREFIX = 'UTILS_TEST_AUTH_LOGIN_';
+    protected const LOGIN_PASSWORD = 'Str0ngPassword!';
```

Comment:
```suggestion
    private const LOGIN_PREFIX = 'UTILS_TEST_AUTH_LOGIN_';
    private const LOGIN_PASSWORD = 'Str0ngPassword!';
```
Za tohle by ti yustme ✂️ 🥚 . To se pouziva zatim jen tady ne?

Reply:
To dědí `SynapseGrantQueryBuilderTest` a mění tu constantu aby tam nebyl konflikt v názvech. Ale LOGIN_PASSWORD može byt private.